runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -2237,9 +2265,7 @@ void init_ofiExchangeAvInfo(void) {
   //
   size_t numAddrs = chpl_numNodes;
   CHPL_CALLOC(ofi_rxAddrs, numAddrs);
-  CHK_TRUE(fi_av_insert(ofi_av, addrs, numAddrs, ofi_rxAddrs, 0, NULL)
-           == numAddrs);
-
+  CHK_TRUE(fi_av_insert(ofi_av, addrs, numAddrs, ofi_rxAddrs, 0, NULL) == numAddrs);  
```

Comment:
Similar question about whether this is related to the touchbar change. But also, we (well, I) try to keep lines under 80 columns, and I think unwrapping this one will make it more than that.

Reply:
My mistake. I added a bunch of print statements while debugging and unwrapped this when removing them.